include/deal.II/lac/la_vector.h

```diff
@@ -110,27 +110,35 @@ namespace LinearAlgebra
     /**
      * Multiply the entire vector by a fixed factor.
      */
-    virtual VectorSpaceVector<Number> &operator*= (const Number factor);
+    virtual Vector<Number> &operator*= (const Number factor);
 
     /**
      * Divide the entire vector by a fixed factor.
      */
-    virtual VectorSpaceVector<Number> &operator/= (const Number factor);
+    virtual Vector<Number> &operator/= (const Number factor);
 
     /**
      * Add the vector @p V to the present one.
      */
-    virtual VectorSpaceVector<Number> &operator+= (const VectorSpaceVector<Number> &V);
+    virtual Vector<Number> &operator+= (const VectorSpaceVector<Number> &V);
 
     /**
      * Substract the vector @p V from the present one.
      */
-    virtual VectorSpaceVector<Number> &operator-= (const VectorSpaceVector<Number> &V);
+    virtual Vector<Number> &operator-= (const VectorSpaceVector<Number> &V);
 
     /**
      * Return the scalar product of two vectors.
      */
-    virtual Number operator* (const VectorSpaceVector<Number> &V);
+    virtual Number operator* (const VectorSpaceVector<Number> &V) const;
+
+    /**
+     * This function is not implemented and will throw an exception.
+     */
+    virtual void import(const ReadWriteVector<Number> &V,
+                        VectorOperation::values operation,
+                        std_cxx11::shared_ptr<const CommunicationPatternBase>
+                        communication_pattern = NULL);
```

Comment:
We usually prefer simply `0` over the `NULL` macro, so I'd suggest you replace them.